daml-lf/interpreter/BUILD.bazel

```diff
@@ -53,6 +53,7 @@ da_scala_test_suite(
         "@maven//:org_scalaz_scalaz_core_2_12",
         "@maven//:org_scalaz_scalaz_scalacheck_binding_2_12",
         "@maven//:org_slf4j_slf4j_api",
+        "@maven//:org_typelevel_paiges_core_2_12",
```

Comment:
Just out of curiosity: What is this about?

Reply:
Because `AnfTest.scala` calls `prettySExpr`. Helpful when debugging the tests.

Reply:
Thanks for the explanation.